galaxy/macros.xml

```diff
@@ -1,31 +1,122 @@
-<?xml version='1.0' encoding='UTF-8'?>
-<!-- CTD2Galaxy depends on this file and on the stdio, advanced_options macros!
-     You can edit this file to add your own macros, if you so desire, or you can
-     add additional macro files using the m/macros parameter -->
-<macros>
-  <xml name="requirements">
-    <requirements>
-      <requirement type="binary">@EXECUTABLE@</requirement>
-    </requirements>
-  </xml>
-  <xml name="stdio">
-    <stdio>
-      <exit_code range="1:"/>
-      <exit_code range=":-1"/>
-      <regex match="Error:"/>
-      <regex match="Exception:"/>
-    </stdio>
-  </xml>
-  <xml name="advanced_options">
-    <conditional name="adv_opts">
-      <param name="adv_opts_selector" type="select" label="Advanced Options">
-        <option value="basic" selected="True">Hide Advanced Options</option>
-        <option value="advanced">Show Advanced Options</option>
-      </param>
-      <when value="basic"/>
-      <when value="advanced">
-        <yield/>
-      </when>
-    </conditional>
-  </xml>
-</macros>
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- CTD2Galaxy depends on this file and on the stdio, advanced_options macros!
+     You can edit this file to add your own macros, if you so desire, or you can
+     add additional macro files using the m/macros parameter -->
+<macros>
+  <token name="@TOOL_VERSION@">3.7</token>
+  <token name="@GALAXY_VERSION@">0</token>
+  <xml name="requirements">
+    <requirements>
+      <requirement type="package" version="@TOOL_VERSION@">python</requirement>
+	  <!--<requirement type="package" version="1.2">ctdopts</requirement>-->
+      <yield/>
+    </requirements>
+  </xml>
+  <xml name="stdio">
+    <stdio>
+      <regex match="Could not allocate metaspace" level="fatal_oom" description="Java memory Exception"/>
+      <regex match="Cannot create VM thread" level="fatal_oom" description="Java memory Exception"/>
+      <regex match="qUncompress: could not allocate enough memory to uncompress data" level="fatal_oom" description="Java memory Exception"/>
+    </stdio>
+  </xml>
+  <xml name="references">
+    <citations>
+      <citation type="doi">doi:10.1186/1471-2105-9-163</citation>
```

Comment:
Will this add OpenMS reference to everything that gets converted from CTD?
Maybe point out, that at least that part should be changed in the macros?